enterprise/api_client/enterprise_catalog.py

```diff
@@ -28,6 +28,9 @@ class EnterpriseCatalogApiClient(UserAPIClient):
     REFRESH_CATALOG_ENDPOINT = ENTERPRISE_CATALOG_ENDPOINT + '/{}/refresh_metadata'
     CATALOG_DIFF_ENDPOINT = ENTERPRISE_CATALOG_ENDPOINT + '/{}/generate_diff'
     ENTERPRISE_CUSTOMER_ENDPOINT = 'enterprise-customer'
+    CONTENT_METADATA_IDENTIFIER_ENDPOINT = (
+        ENTERPRISE_CUSTOMER_ENDPOINT + "/{}/content-metadata/" + "{}"
+    )
```

Comment:
[Question] Why is the string enclosed in a tuple here?

Reply:
Formatter did this 😮‍💨